Singularity/Singularity/Screen/InfoBoxWindow.cs

```diff
@@ -126,6 +131,13 @@ public void Update(GameTime gametime)
             }
         }
 
+        /// <inheritdoc />
+        /// <summary>
```

Comment:
Well, either full (or little) comments or no comments please, not empty comments :)

Reply:
I thought the inheritdoc needed the summary. Apparently not

Reply:
inheritdoc is empty though as well right?

Reply:
I don't think so.. IWindowItem has got a summary for everything